cypress/integration/tests/developer/applicationinventory/reports/filter.test.ts

```diff
@@ -76,26 +76,26 @@ describe("Report Page Filter Validation", { tags: "@tier2" }, () => {
         application.openreport();
 
         // Enter an existing display name substring and assert appName0 exist
-        report.applyFilter(name, application.appName[0].substring(0, 5));
-        cy.get("[role=main]").should("contain.text", application.appName[0]);
+        report.applyFilter(name, application.appName.substring(0, 6));
```

Comment:
For this test case we need multiple applications for filtering . I think we can get multiple applications in source +dependency analysis in which it shows 2 applications . 

So you can try analysisData[1] or 2 .

Reply:
@kpunwatk please review this PR and the comment . 
Source +dep can give us two applications right ?

Reply:
Also if analysisData[9] is not used anywhere else , is it safe to remove it @kpunwatk ?

Reply:
Hi @nachandr @sshveta 
Yes as we can't upload multiple binaries we were not able to test this test cases on Report page.
Great ! yeah source+dependencies gives us "deps and tackle-testapp " we can test this way.
I've this already existing jira task pending https://issues.redhat.com/browse/MTA-146 .
Will refactor and fix this test case.
Thanks @nachandr for looking into it.

Reply:
Thanks @sshveta @kpunwatk . It's been updated to use analysisData[1] .